test/metabase/query_processor_test/date_bucketing_test.clj

```diff
@@ -433,35 +442,62 @@
                 (format "When shifting to %s timezone we should lose %d events and gain %d, for a net gain of %d"
                         timezone-id num-events-gained num-events-lost expected-net-gain))))))))
 
-(def ^:private sad-toucan-events-grouped-by-day
-  ["2015-06-01"
-   "2015-06-02"
-   "2015-06-03"
-   "2015-06-04"
-   "2015-06-05"
-   "2015-06-06"
-   "2015-06-07"
-   "2015-06-08"
-   "2015-06-09"
-   "2015-06-10"])
-
-(defn- results-by-day [parse-fn format-result-fn counts]
-  (map
-   (fn [s cnt]
-     [(-> s parse-fn format-result-fn) cnt])
-   sad-toucan-events-grouped-by-day
-   counts))
-
 (deftest group-by-day-test
   (mt/test-drivers (mt/normal-drivers)
     (testing "\nUTC timezone"
-      (is (= (if (= :sqlite driver/*driver*)
-               (results-by-day u.date/parse date-without-time-format-fn  [6 10 4 9 9 8 8 9 7 9])
-               (results-by-day u.date/parse (format-in-timezone-fn :utc) [6 10 4 9 9 8 8 9 7 9]))
-             (sad-toucan-incidents-with-bucketing! :day :utc))))
+      (do-with-sad-toucan-incidents-with-bucketing!
+       :day :utc
+       (fn [rows]
+         (is (= [["2015-06-01" 6]
+                 ["2015-06-02" 10]
+                 ["2015-06-03" 4]
+                 ["2015-06-04" 9]
+                 ["2015-06-05" 9]
+                 ["2015-06-06" 8]
+                 ["2015-06-07" 8]
+                 ["2015-06-08" 9]
+                 ["2015-06-09" 7]
+                 ["2015-06-10" 9]]
+                rows)))))))
+
+(deftest group-by-day-test-2
+  (mt/test-drivers (mt/normal-drivers)
     (testing "\nPacific timezone"
-      (is (= (cond
-               (= :sqlite driver/*driver*)
+      (do-with-sad-toucan-incidents-with-bucketing!
+       :day
+       :pacific
+       (fn [rows]
+         (is (= (if (or (= :sqlite driver/*driver*)
```

Comment:
Is Sqlite really not covered by either of the below conditions?

Reply:
good point, it is. Fixed